client/src/legacy/LeftAndMain.js

```diff
@@ -772,17 +759,15 @@ $.entwine('ss', function($) {
       var title = xhr.getResponseHeader('X-Title');
       if(title) document.title = decodeURIComponent(title.replace(/\+/g, ' '));
 
-      var newFragments = {}, newContentEls;
+      let newFragments = {};
+      let newContentEls;
       // If content type is application/json (ignoring charset and other parameters)
       if(xhr.getResponseHeader('Content-Type').match(/^((text)|(application))\/json[ \t]*;?/i)) {
         newFragments = data;
       } else {
 
         // Fall back to replacing the content fragment if HTML is returned
-        fragment = document.createDocumentFragment();
-
-        jQuery.clean( [ data ], document, fragment, [] );
-        $data = $(jQuery.merge( [], fragment.childNodes ));
+        $data = $($.parseHTML(data, document, false));
```

Comment:
`jQuery.clean()` is an undocumented function that was removed in 3.0 - but the functionality is effectively the same as `$.parseHTML()`.